editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/document_node_types.rs

```diff
@@ -371,6 +371,35 @@ fn static_nodes() -> Vec<DocumentNodeType> {
 			outputs: vec![DocumentOutputType::new("Image", FrontendGraphDataType::Raster)],
 			properties: node_properties::mask_properties,
 		},
+		DocumentNodeType {
+			name: "Insert Channel",
+			category: "Image Adjustments",
+			identifier: NodeImplementation::proto("graphene_std::raster::InsertChannelNode<_, _, _, _>"),
+			inputs: vec![
+				DocumentInputType::value("Image", TaggedValue::ImageFrame(ImageFrame::empty()), true),
+				DocumentInputType::value("Stencil", TaggedValue::ImageFrame(ImageFrame::empty()), true),
+				DocumentInputType::value("ColorChannel", TaggedValue::RedGreenBlue(RedGreenBlue::Red), false),
```

Comment:
Rename ColorChannel (shown here, and "Target Channel" shown in the Properties panel) to Replace.